tools/check_copyright_and_license_disclaimers.py

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env python
+#
+# Copyright © 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import os
+import sys
+
+
+COPYRIGHT_AND_LICENSE_DISCLAIMER = "# Copyright © 2022 Intel Corporation\n#\n# SPDX-License-Identifier: MIT\n"
+
+def file_has_disclaimer(filename, verbose=False):
+  if verbose:
+    print(f"Checking: {filename}")
+  with open(filename) as fp:
+    # NOTE: allow shebang+empty line before
+    header = "".join(fp.readline() for _ in range(5))
```

Comment:
Does this work if the file is less than 5 lines long?

Reply:
Yeah, it returns empty strings for each of those lines, so then they join as nothing-extra.